### DISCUSSION TITLE
How can I remove "[string "..."]" from my error messages?

### DISCUSSION THREAD

**Discussion by squeezebunny** [mlua-rs/mlua#214] [Q&A] [answered]
I'm loading files as strings, then performing "lua.load()" on those strings. Not sure how else to do it. But I want to clean up my error messages a little more. I would like to present the scripts as just their filenames (which I've started doing with chunk.set_name()) but without the `[string "path/to/file.lua"]:4:...` stuff. Ideally it would show up like `path/to/file.lua:4:...`  Thanks for any help ^^

**Comment by khvzak** [comment, marked as answer]
You can [set chunk name](https://docs.rs/mlua/0.8.4/mlua/struct.Chunk.html#method.set_name) to something like `@file.lua` to cut off `[string "..."]`.
Or just rewrite message using regular expressions / etc.

**Reply by squeezebunny** [reply]
Yup! Adding the @ did it. Thanks a lot! :)
